lib/new_relic/agent/log_event_aggregator.rb

```diff
@@ -230,6 +232,32 @@ def truncate_message(message)
 
         message.byteslice(0...MAX_BYTES)
       end
+
```

Comment:
Rather than continuing to increase the size of this file, I considered pushing the new methods into a `LogLevelFiltering` module and including that in the `LogEventAggregator`. We don't use that pattern a lot in the agent, so I wanted to hold back on that until we could discuss it as a team.

Reply:
I think holding off is a good call. The project wasn't built around a specific pattern that would help guide additional content and it predates RuboCop for that matter. Given the fact that the separation would not promote any reusability throughout the project or other gems and we'd be doing only to keep class sizes down, I think the benefit of compartmentalizing things all in one place for the human reviewer provides some good benefits.

Two things that I like to do to improve things with a large class:

- Make sure there's good `public` and `private` separation
- Have the methods appear in alphabetical order